smallsetup/common/certificate/ZScalerRootCA.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
+MIIE0zCCA7ugAwIBAgIJANu+mC2Jt3uTMA0GCSqGSIb3DQEBCwUAMIGhMQswCQYDVQQGEwJVUzET
```

Comment:
@PraveenGPawadigoudra Folder `smallsetup\common` will be obsolete soon. New configuration and implementations will be under `k2s\lib`. Can we move this certificate close to where it used? eg. `lib\modules\k2s\k2s.node.module\linuxnode\setup\`